Aztec/Classes/Libxml2/DOMString.swift

```diff
@@ -105,15 +131,111 @@ extension Libxml2 {
         }
         
         func setAttributes(_ attrs: [String : Any], range: NSRange) {
+            performAsyncUndoable { [weak self] in
+                self?.setAttributesSynchronously(attrs: attrs, range: range)
+            }
+        }
+        
+        // MARK: - Editing: Synchronously
+        
+        /// Replaces the specified range with a new string.
+        ///
+        /// - Parameters:
+        ///     - range: the range of the original string to replace.
+        ///     - string: the new string to replace the original text with.
+        ///     - inheritStyle: If `true` the new string will inherit the style information from the
+        ///             first position in the specified range.  If `false`, the new sting will have
+        ///             no associated style.
+        ///
+        private func replaceCharactersSynchronously(inRange range: NSRange, withString string: String, inheritStyle: Bool) {
+            rootNode.replaceCharacters(inRange: range, withString: string, inheritStyle: inheritStyle, undoManager: domUndoManager)
+        }
+
+        private func replaceCharactersSynchronously(inRange range: NSRange, withAttributedString attributedString: NSAttributedString, inheritStyle: Bool) {
+            
```

Comment:
I don't want to make whitespace something more relevant than it needs to be.  In this particular case the reason the space is there is to improve readability.

IMHO readability is more important in this case than hard-set rules.